srcStatic/ModuleLoader.cpp

```diff
@@ -67,22 +67,23 @@ void ModuleLoader::RegisterConsoleModules()
 
 void ModuleLoader::RegisterIniModules()
 {
-	const auto sectionNames = GetModuleNames("ExternalModules");
-
-	for (const auto& sectionName : sectionNames)
+	for (const auto moduleName : iniFile.GetKeyNames("ExternalModules"))
```

Comment:
I waffled between ExternalModules and IniModules. Sort of weird to have an IniModules section name in an ini file. Happy to change if @DanRStevens has preference one way or other?

Reply:
Uhh, I don't feel joy about these names, but I don't have a better suggestion right now. Maybe best to look at this again when I'm not tired. I'll open an issue to track.